apps/web-portal-client/lib/features/auth/containers/LoginPage.jsx

```diff
@@ -75,7 +83,9 @@ export function LoginPage() {
               disabled={isLogginInProgress}
               data-test-id="submit"
               className="group relative w-full flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
-              onClick={login}
+              onClick={() => {
+                userLogin({password, email})
```

Comment:
Why not just 'login' ?

Reply:
done, was related to the naming issue in useAuth hook